application/controllers/items.py

```diff
@@ -191,6 +191,8 @@ def delete(self, item_id=None):
         item = Item.query.get(item_id)
         if not item:
             abort(404, description='That item does not exist')
+        elif not item.bids == []:
+            abort(403, description='Cannot delete item with associated bids')
         else:
```

Comment:
We hadn't previously accounted for this situation & API was returning a 500 error. This handles it in a more user-friendly way.